HW9/untitled/src/Office.java

```diff
@@ -0,0 +1,17 @@
+public class Office {
+    /* класс Office, в якого буде поле-масив employees,
+    а також методи, які надають контрольований доступ до внутрішнього стану офісу або інших Employee,
+    наприклад Manager зможе мати додаткові методи, які приймають об'єкт типу Employee на вхід і
+    "змушують" робити його якусь роботу, або методи, які дозволяють керувати офісом
+     */
+
+    private Employee[] employees;
+
+    public Employee getEmployees(){
```

Comment:
`employees` field is private which is good, but we have only a getter, but no setter, how would someone set the value for it?

Also here the array of Employee elements is used without initialization, which means it hold the `null` reference by default so there's a suspicion that without a proper setter method the getter method will always catch NullPointerException trying to execute employees.length for a null reference